applications/ParticleMechanicsApplication/custom_python/add_custom_constitutive_laws_to_python.cpp

```diff
@@ -91,36 +91,36 @@ namespace Python{
     {
         // Linear Elastic laws
         py::class_< LinearElastic3DLaw, typename LinearElastic3DLaw::Pointer, ConstitutiveLaw >
-        (m, "LinearElastic3DLaw").def(py::init<>() )
+        (m, "LinearElasticIsotropic3DLaw").def(py::init<>() )
```

Comment:
Could you name them Particle... because those are occuring also in the structural mechanics Application

Reply:
The structural has one and they name it LinearElastic3DLaw, therefore I changed it to LinearElasticIsotropic3DLaw to avoid problems of running both app. I think adding Particle is not necessary.

Reply:
Alright, I see. Do you know if the CLs are different or if they are the same, just in different applications?

Reply:
I am assuming those were the same and originated from SolidMechanicsApp, but not sure now.